src/api.js

```diff
@@ -138,6 +143,22 @@ function deleteSystemNotifications(baselineId, systems) {
     return postSystemNotifications(path, body);
 }
 
+async function systemFetch(count) {
+    let page = 1;
+    let totalSystems = [];
+    let systems;
+    let systemIds;
+    while (Math.ceil(count / 100) >= page) {
+        let path = `/hosts?per_page=100&page=${page}&order_by=updated&order_how=DESC&staleness=fresh&staleness=stale`;
+        systems = await getSystems(path);
```

Comment:
This does work. But I would suggest initializing API request calls in a loop and then providing those function call references into Promis.all. This will improve the time to fetch those systems as the requests are triggered almost at the same time instead of waiting for each request.

Reply:
I've applied this. thanks for the info.